extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -91,32 +91,40 @@ function render_player( $track_list, $attributes ) {
 	}
 	$instance_id = wp_unique_id( 'podcast-player-block-' );
 
-	$player_data = array(
-		'tracks'     => $track_list,
-		'attributes' => $attributes,
+	// Generate object to be used as props for PodcastPlayer.
+	$player_data = array_merge(
+		// Make all attributes available.
+		$attributes,
+		// And add some computed properties.
+		array(
+			'tracks'   => $track_list,
+			'coverArt' => '',
+		)
 	);
 
 	$block_classname = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attributes );
 
 	ob_start();
 	?>
 	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
-		<ol class="podcast-player__episodes">
-			<?php foreach ( $track_list as $attachment ) : ?>
-			<li class="podcast-player__episode">
-				<a
-					class="podcast-player__episode-link"
-					href="<?php echo esc_url( $attachment['link'] ); ?>"
-					role="button"
-					aria-pressed="false"
-				>
-					<span class="podcast-player__episode-status-icon"></span>
-					<span class="podcast-player__episode-title"><?php echo esc_html( $attachment['title'] ); ?></span>
-					<time class="podcast-player__episode-duration"><?php echo ( ! empty( $attachment['duration'] ) ? esc_html( $attachment['duration'] ) : '' ); ?></time>
-				</a>
-			</li>
-			<?php endforeach; ?>
-		</ol>
+		<noscript>
+			<ol class="podcast-player__episodes">
+				<?php foreach ( $track_list as $attachment ) : ?>
+				<li class="podcast-player__episode">
+					<a
+						class="podcast-player__episode-link"
+						href="<?php echo esc_url( $attachment['link'] ); ?>"
+						role="button"
+						aria-pressed="false"
+					>
+						<span class="podcast-player__episode-status-icon"></span>
+						<span class="podcast-player__episode-title"><?php echo esc_html( $attachment['title'] ); ?></span>
+						<time class="podcast-player__episode-duration"><?php echo ( ! empty( $attachment['duration'] ) ? esc_html( $attachment['duration'] ) : '' ); ?></time>
```

Comment:
We can remove all of these lines for the `<noscript>` section (except the `title` and `duration` lines)

Reply:
Let's keep it for the "HTML fallback" task if you don't mind.